Remotion/Web/Core/ContentSecurityPolicy/CspEnabledHtmlTextWriter.cs

```diff
@@ -91,7 +90,31 @@ public override void RenderBeginTag (HtmlTextWriterTag tagKey)
       base.RenderBeginTag(tagKey);
     }
 
+    public override void AddAttribute (string name, string? value)
+    {
+      ArgumentUtility.CheckNotNull("name", name);
+      AddAttributeWithoutEncodingEventHandlers(name, value, () => { base.AddAttribute(name, value); });
+    }
+
+    public override void AddAttribute (string name, string? value, bool fEndode)
+    {
+      ArgumentUtility.CheckNotNull("name", name);
+      AddAttributeWithoutEncodingEventHandlers(name, value, () => { base.AddAttribute(name, value, fEndode); });
+    }
+
+    protected override void AddAttribute (string name, string? value, HtmlTextWriterAttribute key)
```

Comment:
With the seal-all logic, we can do a simple base call here because the call only forwards the final call. The while extension point is a whitebox implementation so we can do a hard dependency.